internal/proto/controller/storage/auth/oidc/store/v1/oidc.proto

```diff
@@ -163,6 +163,15 @@ message AuthMethod {
     this: "AccountClaimMaps"
     that: "attributes.account_claim_maps"
   }];
+
+  // prompts are the optional prompts allowed for an oidc auth method.
+  // Thee value specifies whether the authorization server prompts
+  // the end-user for reauthentication, account selection and consent
+  // @inject_tag: `gorm:"-"`
+  repeated string prompts = 220 [(custom_options.v1.mask_mapping) = {
+    this: "prompts"
+    that: "attributes.prompts"
+  }];
```

Comment:
I think, we need a `message Prompt` as well.  See one of the other value objects
like signing algs as an example.